### PR TITLE
docs: app-name not coming on browser

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #33612 

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -669,7 +669,7 @@ In `angular.json` add two new configuration sections under the `build` and `serv
      ...
     },
     "es5": {
-      "browserTarget": "<app-name>:build:es5"
+      "browserTarget": "'<app-name>':build:es5"
```
Comment (gkalpak) [inline aio/content/guide/deployment.md]: The correct way to fix this is to replace `<`/`>` with HTML entities:

```diff
&lt;app-name&gt;
```

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/deployment.md (hunk 1 above)
Comment (santoshyadav198613) [reply aio/content/guide/deployment.md]: ahh, cool will make the changes :)

**Comment by kapunahelewong** [comment]
Thanks for this, @santoshyadav198613. Could you add `Fixes #33612` to the commit message like this:

```
docs: app-name not rendering on browser

Fixes #33612 

```
Please tag me when you're ready. Thanks!

Link so it'll show up in issue: https://github.com/angular/angular/issues/33612

**Comment by santoshyadav198613** [comment]
Hi @kapunahelewong ,
Made the changes.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Pull request closed without merging**